packages/inter-protocol/scripts/init-core.js

```diff
@@ -71,15 +71,16 @@ const installKeyGroups = {
  * @param {(m: string, b: string, opts?: any) => I} opts.install
  * @param {<T>(f: T) => T} [opts.wrapInstall]
  *
+ * @param options
  * @template I
  * @template R
  */
-export const committeeProposalBuilder = async ({
-  publishRef,
-  install: install0,
-  wrapInstall,
-}) => {
-  const { ROLE = 'chain' } = process.env;
+export const committeeProposalBuilder = async (
+  { publishRef, install: install0, wrapInstall },
+  options = {},
+  { env = process.env } = {},
+) => {
+  const { ROLE = env.ROLE || 'chain', econCommitteeOptions } = options;
```

Comment:
better to pass `process.env` than rely on ambient authority, but why pass it at all if it can come in through `options.ROLE`? Multiple ways to express the same configuration makes the code harder to follow. It's not apparent without reading `committeeProposalBuilder` that `env.ROLE` is valid or takes precedence over `options.ROLE`.

Reply:
Turns out `ROLE` was unnecessary.  Removed.

Reply:
Seems to still be there as of 2a2b2881946288ad0c7191